services/workflows-service/src/workflow/dtos/document-update-update-input.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsObject } from 'class-validator';
+
+export class DocumentUpdateInput {
+  @ApiProperty({
+    required: true,
+    type: Object,
+  })
+  @IsObject()
+  document!: any;
```

Comment:
Until Typebox is in for documents.
```suggestion
document!: Record<string, unknown>;
```